src/darwin/Framework/CHIP/MTRDeviceControllerDataStore.mm

```diff
@@ -26,6 +26,9 @@
 #include <lib/support/CodeUtils.h>
 #include <lib/support/SafeInt.h>
 
+// Log error each time sync storage takes longer than this threshold
+#define SYNC_OPERATION_DURATION_LOG_THRESHOLD (2.0)
```

Comment:
it may be desirable to add a unit of measurement for these  like _SEC or _MS.

Reply:
Done.